### PR TITLE
Portfolio setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by Banstein**
Add Html file
Add Css file
Add Linters.yml
Add .gitignore
Add .hintrc
Add .stylelintrc
Add README.md
Add background image.png
Create LICENSE
Create package-lock.json
Create package.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
+# Portfolio
```
Comment (lily-coder) [inline README.md:1]: - [ ] [REQUIRED] Please ensure you add a short description of your project. This is to help the users get an overview of what the project is about.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="logo">My Logo</div>
```
Comment (lily-coder) [inline index.html]: - [ ] [REQUIRED] Please ensure that the logo has a link (a tag). If you look at most websites, the logo navigates users to the home page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <nav class="menu"><span class="material-icons">menu</span></nav>
+    </header>
+    <main>
+        <section class="main-content">
+            <h2 class="intro-text">I'm Islam Glad to see you!</h2>
```
Comment (lily-coder) [inline index.html]: - [ ] [REQUIRED] Please ensure this main title of the page is a h1 tag. This ensures good SEO

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hi @Banstein , 

Good job so far!🎉🎉🎉

To Highlight! 🟢
✔️ Good commit messages
✔️ Good job with the linter checks
✔️ Correct Github flow was used

There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Banstein) [reply index.html]: Okay all Done!

File: README.md [hunk 4]
```diff
@@ -1 +1,47 @@
...
+Hi. I’m Islam, a Full-stack Developer I'm showing my experience of creating successful websites.
```
Comment (keneogu) [inline README.md]: - [ ] Please add the description of the project not a summary of who you are. e.g "Mobile version and setup of the figma design"

File: index.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+            <p class="main-text"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </section>
+        <section class="footer">
+            <h4 class="end-text">LETS'S CONNECT</h4>
+            <ul class="social-conacts">
+                <li><i class="fa fa-twitter"></i></li>
+                <li><i class="fa fa-linkedin"></i></li>
+                <li><i class="fa fa-medium"></i></li>
+                <li><i class="fa fa-github"></i></li>
+                <li><i class="fa fa-angellist"></i></li>
```
Comment (keneogu) [inline index.html]: - [ ] I suggest you make the icons a link using the `<a>` tag. This makes it easier for anyone or companies to get in contact with you.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @Banstein  , 

Great work on making the changes requested by a previous reviewer 👏 

- Linters working properly ✔️
- Well documented PR description ✔️
- Commit messages are meaningful ✔️
- Good HTML structure ✔️

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------

**Review by lily-coder — APPROVED** [review]
Hi @Banstein , 

Good job implementing all changes as requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**